src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain;
+
+import lotto.exception.InvalidLottoNumberException;
+
+import java.util.Objects;
+
+public class LottoNumber {
+
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private final int value;
+
+    public LottoNumber(int value) {
+        validateValue(value);
+        this.value = value;
+    }
+
+    private void validateValue(int value) {
+        if (value < MIN_LOTTO_NUMBER || value > MAX_LOTTO_NUMBER) {
+            throw new InvalidLottoNumberException();
+        }
+    }
+
+    public int getValue() {
+        return value;
+    }
```

Comment:
```suggestion
    public int toInt() {
        return value;
    }
```

이건 개인 취향이기는 한데, 저는 내부의 멤버를 직접적으로 가져오는 getter 함수 보다는,
LottoNumber 타입을 Integer 타입으로 "형변환" 한다는 느낌을 좋아합니다.
기왕 원시값을 불변 객체로 포장했다면, 그 객체 자체를 value 로 다룬다는 느낌은 어떨까 제안해봅니다.